### PR TITLE
Ensure you have apt-transport-https

### PR DESCRIPTION
**Pull request by oscarmlage**
Before to make an update you should ensure you have the apt-transport-https installed

### CODE REVIEW

**Comment by saghul** [comment]
LGTM, thanks! Have you signed the CLA?

**Comment by oscarmlage** [comment]
@saghul Yep, I've just signed the CLA right now.

File: doc/quick-install.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ echo 'deb https://download.jitsi.org stable/' >> /etc/apt/sources.list.d/jitsi-s
 wget -qO -  https://download.jitsi.org/jitsi-key.gpg.key | apt-key add -
 ```
 
+### Ensure you have apt-transport-https installed
+
+Before to make an update you should ensure you have the apt-transport-https installed
```
Comment (saghul) [inline doc/quick-install.md]: "Before making an update"

Comment (saghul) [inline doc/quick-install.md]: Also please finish the sentence with a period.

Comment (oscarmlage) [inline doc/quick-install.md]: Thanks for the corrections @saghul, how should I proceed to correct the errors? new PR?

Comment (damencho) [inline doc/quick-install.md]: You can amend the commit in your local branch and the force push to your repo.

**Pull request closed without merging**